src/usr/local/captiveportal/index.php

```diff
@@ -189,11 +189,11 @@
 			$passwd = $cpcfg['radmac_secret'];
 			$context = 'radmac'; // Radius MAC authentication
 		} elseif (!empty($_POST['auth_user2'])) { 
```

Comment:
this shouldn't be better like that ?
`} elseif (!empty(trim($_POST['auth_user2']))) { `
because : empty(" ") is false